### ISSUE TITLE
[feat-req] add exempt project

### ISSUE THREAD

**Issue by jd-0001** [actions/stale#650]
## The problem

We have issues which are `feat-req` but some of them are considered but others are not.

Features that are considered will be moved to a project named `unplanned` so it doesn't get closed.

Although we can achieve it using adding a new label but it might be called a workaround and not a solid solution.

## The solution

Add exempt project option.

**Comment by C0ZEN** [comment]
@jd-0001 This action https://github.com/Sonia-corporation/stale support such feature through `issue-ignore-all-project-cards` which ignore any issue which has a project card associated to it.

**Comment by jd-0001** [comment]
Yes, I found it while I was searching for this feature in this repo.

I prefer this action to provide this feature as this is by the GitHub team and rules other than projects are already implemented.

Thanks for your suggestion :)

**Comment by C0ZEN** [comment]
There is no active contributor on this repo any more and the feature, as far as I know, was not possible to implement due to the usage of the REST API.
The one I suggested is using GraphQL and so have way more possible features.

**Comment by leighmcculloch** [comment]
This would be very useful for the @stellar repos as well. We often plan longer term work by moving issues onto project boards and it is annoying to get stale notifications on those issues or have to label those issues. It would be convenient if we could simply state that any issue on a project board should be ignored.

**Comment by jd-0001** [comment]
keep it open

**Comment by jd-0001** [comment]
keep it open

**Comment by jd-0001** [comment]
.

**Comment by jd-solanki** [comment]
.

**Comment by jd-solanki** [comment]
.

**Comment by jd-solanki** [comment]
.

**Comment by juliajohannesen** [comment]
Worth noting that the alternative that offers this feature, [`Sonia-corporation/stale`](https://github.com/Sonia-corporation/stale) appears to be unmaintained (last commit ~2023), and no longer works due to using the Projects v1 API, [which is being sunset](https://github.blog/changelog/2024-05-23-sunset-notice-projects-classic/). Adding the feature to include/exclude issues and PRs that are on specific project boards would enable a migration path for users of that action.
